app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if command.split()[1] == command.split()[2]:
+        return None
```

Comment:
no need som many split, you can unpack values from the command with one split().
 `variable, variable, variable = command.split()`